### PR TITLE
Updated FindWwise.cmake to check Wwise version

### LINKED ISSUE #4945: FindWwise.cmake uses incompatible version of Wwise if multiple Wwise versions installed
**Describe the bug**
A clear and concise description of what the bug is. Try to isolate the issue to help the community to reproduce it easily and increase chances for a fast fix.
If older version of wwise is installed, cmake uses older version instead of the proper version.

**Steps to reproduce**
1. Lumberyard installed with wwise.
2. Follow the instruction in Setup page.
3. At the build process, it stops while building the sound gem.

**Expected behavior**
Use proper version of wwise for build the project.

**Actual behavior**
Grab older version of the wwise.

**Assets required**
N/A

**Screenshots/Video**
If applicable, add screenshots and/or a video to help explain your problem.

**Found in Branch**
develop branch.[74d7405]

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows 10
 - Version: 21H1
 - CPU Ryzen 3850X
 - GPU Nvidia GTX 2070super
 - Memory: 32GB

**Additional context**
Posted to the discord channel of https://discord.com/channels/805939474655346758/861713485725433857/900844680089071717
and get quick solution to use LY_WWISE_INSTALL_PATH 

Ideally, FindWise.cmake should validate the version.

### PR DESCRIPTION
**Pull request by amzn-phist**
- Reads version information from a header file in the SDK.
- Compares it against a "minimum" SDK version and prints versions if
  there's no supported version available to use.

closes #4945

### CODE REVIEW

**Review by lumberyard-employee-dm — APPROVED** [review]
I am assuming you tested this out by setting WWISEROOT to an older Wwise SDK?

**Comment by amzn-phist** [comment]
> I am assuming you tested this out by setting WWISEROOT to an older Wwise SDK?

Well, I tested the error path by setting the minimum version to be something higher than what I have installed.  That seemed the easier way because 1) I only have one version installed right now, and 2) I still needed WWISEROOT to point to something valid.

**Comment by lumberyard-employee-dm** [comment]
> > I am assuming you tested this out by setting WWISEROOT to an older Wwise SDK?
> 
> Well, I tested the error path by setting the minimum version to be something higher than what I have installed. That seemed the easier way because 1) I only have one version installed right now, and 2) I still needed WWISEROOT to point to something valid.

Works for me.

**Review by AMZN-alexpete — APPROVED** [review]

**Review by FiniteStateGit — APPROVED** [review]
Thanks for the help on this!

File: cmake/3rdParty/FindWwise.cmake [hunk 1]
```diff
@@ -9,13 +9,33 @@
 # Wwise Install Path
 set(LY_WWISE_INSTALL_PATH "" CACHE PATH "Path to Wwise installation.")
 
-# Check for a known file in the SDK path to verify the path
+# Wwise Version
+set(WWISE_VERSION)
+set(MINIMUM_WWISE_VERSION "2021.1.1.0")
```
Comment (amzn-changml) [inline cmake/3rdParty/FindWwise.cmake:14]: Our build nodes meet should meet min version, but best to check this out in your AR branch build.

**Review by amzn-changml — APPROVED** [review]

File: cmake/3rdParty/FindWwise.cmake (hunk 1 above)
Comment (amzn-phist) [reply cmake/3rdParty/FindWwise.cmake:14]: This Gem isn't built on AR due to the Wwise SDK dependency.

**Pull request merged**